conans/client/generators/cmake_common.py

```diff
@@ -111,15 +113,20 @@ def cmake_global_vars(deps, build_type=""):
 
 _target_template = """
     conan_package_library_targets("${{CONAN_LIBS_{uname}}}" "${{CONAN_LIB_DIRS_{uname}}}"
-                                  CONAN_PACKAGE_TARGETS_{uname} "{deps}" "" {pkg_name})
+                                  CONAN_PACKAGE_TARGETS_{uname} "${{CONAN_SYSTEM_LIBS_{uname}}} {deps}"
+                                  "" {pkg_name})
     conan_package_library_targets("${{CONAN_LIBS_{uname}_DEBUG}}" "${{CONAN_LIB_DIRS_{uname}_DEBUG}}"
-                                  CONAN_PACKAGE_TARGETS_{uname}_DEBUG "{deps}" "debug" {pkg_name})
+                                  CONAN_PACKAGE_TARGETS_{uname}_DEBUG "${{CONAN_SYSTEM_LIBS_{uname}_DEBUG}} {deps}"
+                                  "debug" {pkg_name})
```

Comment:
why do we need "debug" here and below now?

Reply:
What do you mean? This is only adding `CONAN_SYSTEM_LIBS_<NAME>_DEBUG` to the list of the dependencies to link with

Reply:
yeah, I see now, sorry for the noise :)